lib/node/spvnode.js

```diff
@@ -275,15 +275,20 @@ class SPVNode extends Node {
   }
 
   /**
-   * Broadcast a transaction (note that this will _not_ be verified
-   * by the mempool - use with care, lest you get banned from
-   * bitcoind nodes).
+   * Add transaction to txFilter, broadcast (note that this will
+   * _not_ be verified by the mempool - use with care, lest you
+   * get banned from bitcoind nodes).
    * @param {TX} tx
    * @returns {Promise}
    */
 
-  sendTX(tx) {
-    return this.broadcast(tx);
+  async sendTX(tx) {
```

Comment:
To protect users from getting banned by sending an invalid tx, would it make sense to first check `tx.verify()` before anything else?

Reply:
Fixed following the full node example: https://github.com/bcoin-org/bcoin/blob/f585d8616df8335fddbc21d1b776a979ec6ff305/lib/mempool/mempool.js#L785-#788
https://github.com/bcoin-org/bcoin/blob/f585d8616df8335fddbc21d1b776a979ec6ff305/lib/node/fullnode.js#L297-#303